### ISSUE TITLE
[Discussion] Describe linter exceptions in dedicated configuration file

### ISSUE THREAD

**Issue by wuzzapcom** [golangci/golangci-lint#864]
Hi everyone. I am looking for linter integration for my project. Currently I want to wrap linter so many developers could easy integrate linter to their projects.
For me linter's the biggest problem is false positive. And I can imagine that many people may do not like to insert some tool-related comments to their valuable source code.
Have you tried to use some kind of configuration files for this? Is this idea healthy?
For example:
```yaml
exceptions:
- internal/package/file.go:120:
  - gosec
```
Of course this config will expire with active development. But linter may detect this expired messages and fail on them too. So code in master will always contain the linter-passing code and actual exception config file.

What do you think about such idea? I will be happy to listen to your opinion as you have more expertise then me.

**Comment by maratori** [comment]
You can use `issues` section in config file: https://github.com/golangci/golangci-lint#config-file
